shifthelper/tools/shift.py

```diff
@@ -9,8 +9,19 @@
 log = logging.getLogger(__name__)
 
 
-
-def whoisonshift(clear_cache=False):
+calendar_query = '''
```

Comment:
But multiline strings in indented code are kind of ugly :D 

But that's not a real reason.